### ISSUE TITLE
initialize method never called without setting fx:controller in fxml

### ISSUE THREAD

**Issue by mtbadi39** [roskenet/springboot-javafx-support#66]
Without setting `fx:controller="someController"` in the **.fxml** file, the initialize method never called !
So what the utility of the `@FXMLController` annotation.

**Comment by roskenet** [comment]
This is true. This annotation includes (as you see in the code) at the moment only the @Component annotation, to 'component scan' it as a Spring Bean. We "invented" this annotation to 
a) have a bit of semantic
b) be safe in the future to add some fields there.

**Comment by roskenet** [comment]
Feel free to implement a convention for 'auto detecting' a controller from a bean. (In fact we haven't done this, because we could not agree on whether this should be called XYZController or XYZPresenter) ;-)
